dali/base/dadfs.cpp

```diff
@@ -1396,15 +1398,24 @@ class CDelayedDelete: public CInterface
                 {
                     case SDSExcpt_LockTimeout:
                     case SDSExcpt_LockHeld:
-                        e->Release();
+                        timeoutException.setown(e);
                         break;
                     default:
                         throw;
                 }
             }
             file.clear();
+            unsigned remaining;
+            if (timer.timedout(&remaining))
+            {
+                StringBuffer timeoutText;
```

Comment:
Should you release the exception ?  As in:
```
StringBuffer timeoutText;
TimeoutException->errorMessage(timeoutText);
TimeoutException->Release();
throwStringExceptionV(-1, "Failed to remove %s: %s", logicalname, timeoutTest.str());
```

Reply:
That would cause a double release and crash, since timeoutException is Owned.a

Reply:
Yes, sorry I missed seeing that it is Owned.